### DISCUSSION TITLE
Add Azure Poland Central region

### DISCUSSION THREAD

**Discussion by taurit** [VarunAgw/CloudPingTest#6] [Ideas] [answered]
Hi,

Azure recently launched a new region in Europe, `Poland Central`. The documentation is [lagging a bit](https://azure.microsoft.com/en-us/explore/global-infrastructure/geographies/#geographies) and still shows "coming soon", but perhaps it could be added to a ping test tool already?

It would be nice to compare the latencies in this new region to those in other European data centers.

PS I love the latency test tool. Thanks! :)

**Comment by VarunAgw** [comment, marked as answer]
Added
